net-libs/libwebsockets/metadata.xml

```diff
@@ -19,12 +19,10 @@
 		libwebsockets is a C library designed to make development of websockets clients and servers easier (possible).
 	</longdescription>
 	<use>
-		<flag name="ssl">Include SSL support (default <pkg>dev-libs/openssl</pkg>)</flag>
-		<flag name="zlib">Include zlib support</flag>
-		<flag name="static-libs">Build the static version of the library</flag>
+		<flag name="client">The client part of the library</flag>
```

Comment:
This is what is written in  its `CMakeLists.txt` https://github.com/warmcat/libwebsockets/blob/master/CMakeLists.txt
It's supposed to be a testing client as described in https://github.com/warmcat/libwebsockets/blob/master/README.test-apps.md
 @lws-team could you provide the word to be here to describe "client"? thanks :)